src/components/InstanceMonitoring/InstanceMonitoringMetric.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InstanceMonitoringMetric = ({ amount, unit, desc }) => (
+  <li>
+    <span className="infra--instance-amount">{amount}</span>
+    <span className="infra--instance-unit">{unit}</span>
+    <br />
+    <span className="infra--instance-desc">{desc}</span>
+  </li>
+);
+
```

Comment:
Could you please add `propTypes` (with comments for each props)? Thanks!

Reply:
Also @asudoh could you give me an example in the code how this is currently done? thanks!

Reply:
Hi @chrsfshr here’s an example: https://github.com/IBM/carbon-components-react/blob/5e13b23/src/components/TooltipIcon/TooltipIcon.js#L30-L46 - Let me know if you have any questions or I’m missing anything. Thanks!